tests/integration/modules/test_cp.py

```diff
@@ -481,7 +482,8 @@ def test_hash_file(self):
                 [
                     'salt://grail/scene33',
                 ])
-        with salt.utils.fopen(path, 'r') as fn_:
+        mode = 'rb' if salt.utils.is_windows() else 'r'
+        with salt.utils.fopen(path, mode) as fn_:
             data = fn_.read()
             if six.PY3:
                 data = salt.utils.to_bytes(data)
```

Comment:
Well, the salt master who has the original hash is also on windows so, line endings would match. Its something else...